conda/shell/etc/profile.d/conda.sh

```diff
@@ -35,7 +42,7 @@ conda() {
             ;;
         install|update|upgrade|remove|uninstall)
             __conda_exe "$@" || \return
-            __conda_activate reactivate
+            __conda_reactivate
```

Comment:
We can leave this one, but we do need to export __conda_reactivate so mamba 1.x can use it.